client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/Forms/EilatOrDeadSea.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { Controller, useFormContext } from 'react-hook-form';
+
+import Toggle from 'commons/Toggle/Toggle';
+import FormTitle from 'commons/FormTitle/FormTitle';
+import { fieldsNames } from 'commons/Contexts/ExposuresAndFlights';
+import FormRowWithInput from 'commons/FormRowWithInput/FormRowWithInput';
+
+import useStyles from '../ExposuresAndFlightsStyles';
+
+
+interface Props {
+    wasInEilat: boolean;
+    wasInDeadSea: boolean;
+}
+
+export const EilatOrDeadSea = (props: Props) => {
+	const { control } = useFormContext();
+
+    const { wasInEilat , wasInDeadSea} = props;
+    const classes = useStyles();
+
+    return (
+		<div className={classes.subForm}>
```

Comment:
fix indentation for all the component

Reply:
https://github.com/github/VisualStudio/issues/1524#issuecomment-372632866
omg , this apperars to be an issue with Github in general! , it happens when the file is indented with tabs!

Reply:
fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1017/commits/6d97c624f2c7984a485b6f723ad9258310340e96)